nextflow.config

```
@@ -100,8 +100,10 @@ params {
     max_memory                 = '128.GB'
     max_cpus                   = 16
     max_time                   = '240.h'
+
     // Juicer Java paramaters
-    juicer_jvm                 = '-Xms512m -Xmx4096m'
+    juicer_jvm_params          = '-Xms512m -Xmx4096m'
+    juicer_tools_jar           = null
```

Can we set this to the remote web URL so that by default it downloads it from the web?

I think that's what @mahesh-panchal was suggesting here: https://github.com/nf-core/hicar/pull/2#issuecomment-958949989

```suggestion
    juicer_tools_jar           = 'https://github.com/aidenlab/Juicebox/releases/download/v2.13.06/juicer_tools_2.13.06.jar'
```